docs/docs/rest-api/public/api/v2/types/volumes.raml

```diff
@@ -0,0 +1,70 @@
+#%RAML 1.0 Library
+uses:
+  label: label.raml
+  strings: stringTypes.raml
+types:
+  Volume:
+    type: object
+    properties:
+      name:
+        type: strings.Name
+        description: The name of the volume to reference.
+      host?:
+        type: string
```

Comment:
using a simple type here makes it harder to add host-volume properties later. consider making this an object type with `path` property